### PR TITLE
[ENG-2946][ENG-3003][ENG-3004] Fix a11y problems on discover and landing pages

### PR DESCRIPTION
**Pull request by brianjgeiger**
## Purpose
Fixes a11y issues on the discover and landing pages

## Summary of Changes/Side Effects
1. Add `lang=eng-us` to HTML tag
2. Add `role=option` to options inside carousel
3. Add aria-label to disclosure triangles for subjects
4. Update version of ember-osf to fix aria-label for search box on discover page
5. Add yalc-specific gitignores
6. Fix aria-label on search input box for preprints landing page
7. Add language to translations file
8. Remove `<small>` tags from 'Browse by' language
Screen Shot 2021-08-02 at 11 58 30 AM

9. Darken background image for increased contrast
Screen Shot 2021-08-02 at 11 46 14 AM

## Testing Notes
This will need to be run through the selenium AXE tests again to see if these fixes did what they were supposed to. Nothing should be broken as far as functionality goes.

## Ticket

https://openscience.atlassian.net/browse/ENG-2946
https://openscience.atlassian.net/browse/ENG-3003
https://openscience.atlassian.net/browse/ENG-3004

## Notes for Reviewer
This will need to be tested with https://github.com/CenterForOpenScience/ember-osf/pull/453

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

**Review by fabmiz — APPROVED** [review]

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
One minor fix to get the role=button and aria-label picked up properly

File: app/templates/components/search-facet-taxonomy.hbs [hunk 1]
```diff
@@ -2,7 +2,12 @@
     <ul>
         {{#each topLevelItem as |item|}}
             <li>
-                <span role="button" class="pointer m-r-xs {{if item.showChildren 'taxonomy-show-children'}}" {{action 'expand' item}}>
+                <span
+                    role="button"
```
[the remark this answers is not in the document]
Comment (brianjgeiger) [reply app/templates/components/search-facet-taxonomy.hbs:6]: Fixed

**Review by futa-ikeda — DISMISSED** [review]
🚀

**Review by fabmiz — COMMENTED** [review]
@brianjgeiger, we need to re-yarn because of the `ember-osf` dependency update

**Comment by brianjgeiger** [comment]
@fabmiz Done, and tests are passing.

**Pull request merged**